packages/components/package.json

```
@@ -35,6 +35,7 @@
 		"classnames": "^2.2.5",
 		"clipboard": "^2.0.1",
 		"dom-scroll-into-view": "^1.2.1",
+		"hex-rgb": "4.1.0",
```

Using ranges is safer for usage from npm:
```suggestion
		"hex-rgb": "^4.1.0",
```

Otherwise, projects may end up bundling two different versions of the same library.

---

Will do! Thanks!